db/fixtures/ae_datastore/ManageIQ/ConfigurationManagement/AnsibleTower/Service/Provisioning/StateMachines/Provision.class/__methods__/post_provision.rb

```diff
@@ -0,0 +1,12 @@
+#
+# Description: This method examines the AnsibleTower job provisioned
+#
+$evm.log("info", "Starting Ansible Tower Post-Provisioning")
+
+service = $evm.root["service_template_provision_task"].destination
```

Comment:
@bzwei 
Can we check if we have a service_template_provision_task and service object before we call service.job